src/tests/Signup/index.test.jsx

```diff
@@ -28,4 +28,18 @@ describe('Sign up Component', () => {
     wrapper.find('#firstName').simulate('change', event);
     expect(spy).toHaveBeenCalledTimes(1);
   });
+
+  it('should handle text lastname input', () => {
+    const event = {
```

Comment:
How is this related to the ticket?

Reply:
The test coverage drop was arising from the Signup component so I worked on raising the test coverage.

Reply:
Can you use new tickets and pull requests for cases of increasing coverage?

Reply:
Alright, I will do that.

Reply:
Remember to remove this change when you are done with that.

Reply:
Alright.